### ISSUE TITLE
Not working with Android 9 Pie?

### ISSUE THREAD

**Issue by Spleeeno** [miracle2k/android-autostarts#31]
Hi there,
just upgraded one of my devices from Oreo to Pie - LineageOS, clean install.
According to some playstore reviews - and my own experience after upgrading - Autostarts doesnt work on Android 9.
So is this still actively maintained? Any way to fix that? Would be greatly appreciated ;)

I thought that giving a bad review on the store after years of using this as one of the essential apps (on every of my rooted devices) was a bit of a dick move, so instead I opened this issue...

**Comment by AreYouLoco** [comment]
Reference: https://github.com/miracle2k/android-autostarts/issues/23

**Comment by Spleeeno** [comment]
Kinda doubt the referenced Issue is relevant since Autostarts ran fine on Android Nougat, which was new back in 2016. Or if it didnt it was fixed with 1.9.8, it's been a while...
https://github.com/miracle2k/android-autostarts/blob/master/CHANGES

I'm just saying it stopped working with Android Pie - and that one was released in 2018.

**Comment by T-vK** [comment]
I would really like to see an update. I've been using Autostarts for many years to work around the issue that firewalls on Android don't work for the first few seconds after Android has booted. Not sure what I'm gonna do now...

**Comment by STPKITT** [comment]
I agree, this apps is awesome and essential so it would be great to see it updated and also built on F-Droid since that's my only app source (pay via Google, install via F-Droid).

**Comment by Gitoffthelawn** [comment]
@T-vK I recommend putting any Android device into Airplane Mode before rebooting.  Unfortunately, this isn't possible if the device runs out of battery or crashes.

**Comment by DJiK** [comment]
+1

**Comment by Drugoy** [comment]
Does it work with newer versions of Android?
